cfme/common/provider_views.py

```diff
@@ -94,16 +94,25 @@ def __getattribute__(self, item):
 
     @property
     def is_displayed(self):
-        if self.context['object'].appliance.version >= '5.7':
-            subtitle = 'Summary' if self.toolbar.view_selector.selected == 'Summary View' \
-                else 'Dashboard'
-        else:
-            subtitle = 'Summary'  # 5.6 has only only Summary view
-        title = '{name} ({subtitle})'.format(name=self.context['object'].name, subtitle=subtitle)
+        subtitle = 'Summary' if self.toolbar.view_selector.selected == 'Summary View' \
+            else 'Dashboard'
```

Comment:
**REQUIRED**
Let's try to avoid ternary operator and `\`
```python
if self.toolbar.view_selector.selected == 'Summary View':
    subtitle = 'Summary'
else:
    subtitle = 'Dashboard'